src/annotator/anchoring/pdf.js

```diff
@@ -448,55 +480,51 @@ export function anchor(root, selectors) {
  * @param {HTMLElement} root - The root element
  * @param {Range} range
  */
-export function describe(root, range) {
-  const normalizedRange = new BrowserRange(range).normalize();
+export async function describe(root, range) {
+  // "Shrink" the range so that the start and endpoints are at offsets within
```

Comment:
In the previous variant, with `BrowserRange`—did `BrowserRange.normalize` have the equivalent result?

Reply:
For the purposes of this function the result was equivalent. There is a difference in that `normalize(...)` would produce a `NormalizedRange` which always starts at the beginning of a text node and ends at the end of a text node, which may require splitting existing text nodes. `normalize(...)` also did not clearly define what happens if there were no text nodes within the range. The new code on the other hand returns a range which will start and end within text nodes, but not necessarily at the start. This means that it will not need to modify the DOM in the process. It is also better defined what will happen if the range contains no text.